packaging/greenboot/microshift-running-check.sh

```diff
@@ -25,6 +25,36 @@ function forced_termination() {
     exit 1
 }
 
+# Check preconditions for existence of topolvm deployment.
+# Adapted from MicroShift code.
+#
+# args: None
+# return: 0 if topolvm readiness should be checked, 1 otherwise
+function topolvmShouldBeDeployed() {
+    if ! hash vgs 2>/dev/null; then
+        return 1
+    fi
+
+    # TODO: If lvmd.yaml exists, should we expect topolvm even if the OS doesn't has lvms2 installed?
+    if [ -f /etc/microshift/lvmd.yaml ]; then
+        return 0
+    fi
+
+    local -r volume_groups=$(vgs --readonly --options=name --noheadings)
+    if [ -z "${volume_groups}" ]; then
+        return 1
+    fi
+
+    local -r volume_groups_count=$(echo "${volume_groups}" | wc -l)
+    if [ "${volume_groups_count}" -eq 1 ]; then
+        return 0
+    elif echo "${volume_groups}" | grep -q "microshift"; then
```

Comment:
Do we expect the volume to be called `microshift` exactly, or container `microshift` in the name?

Reply:
We expect volume group to be named `microshift`
https://github.com/openshift/microshift/blob/682d0d580b6d568915baa67188d918428837ad65/pkg/config/lvmd/lvmd.go#L103
https://github.com/openshift/microshift/blob/682d0d580b6d568915baa67188d918428837ad65/pkg/config/lvmd/lvmd.go#L57

Reply:
In this case, let's perform an exact comparison.
```suggestion
    elif echo "${volume_groups}" | grep -qw "microshift"; then
```

Reply:
Done